### ISSUE TITLE
QR logins should be easy to find 

### ISSUE THREAD

**Issue by CouldBeThis** [nextcloud/android#6877]
Hello,

I hope I have found the right place to post this. 

### Is your feature request related to a problem? Please describe.

I installed the Nextcloud app from the google play store. First thing needed is to login to my server. I see there is a QR option which is *fantastic* because otherwise I need to generate the credentials on my desktop and tediously get the to me phone. But then... where does the QR code come from? There is no help text, no instruction, link to a support page, no nothing. 

So back on the desktop I am searching. Found this thread from Feb 2019 describing: [Cannot find QRCode for setting up mobile clients](https://help.nextcloud.com/t/cannot-find-qrcode-for-setting-up-mobile-clients/48202)

>  it looks like the mobile app has QRCode scanner built in for easier client setup, but I cannot find where in my Nextcloud server are those QRCode for client setup

(There is also links to gh issues I cannot understand; I think it was about the technical implementation of the QR feature.) 

### Describe the solution you'd like

Since the above was in a  support thread, a very helpful explanation was [provided](https://help.nextcloud.com/t/cannot-find-qrcode-for-setting-up-mobile-clients/48202):

> Reiner_Nippes Jun '19
> 
> settings -> user security -> generate new password
>  
> ![](https://help.nextcloud.com/uploads/default/original/3X/c/9/c954f65c58437ce9b6ba95353fbd9d2c6347b05d.png)
> 
> grafik
> 
> the text “QR-Code für mobile Apps anzeigen” is a link. click it.

This information should be provided

1. In the app either directly on the screen with the QR code, AND
   - Could link to  a help page instead if clutter is undesirable 
   - *but* if the QR code is easier to get to then instructions would fit easily on a phone screen. 
   - personally I like to have the information I need available where I need it
3. The text of the above method (generating app password) should indicate that there is a QR code. Even though I have made ap passwords many times, I was still unable to recall that this is where the option lives. An tiny QR code icon would help attract the eye. 
![200902nexclourqrcode2](https://user-images.githubusercontent.com/53547181/91943461-a096ae80-ecca-11ea-8a1b-c1cd277becb8.png) AND
2. In the bottom left corner thing, like this (sorry it's so crappy, just to show the concept):
![200902nexclourqrcode](https://user-images.githubusercontent.com/53547181/91942556-231e6e80-ecc9-11ea-9c73-7f464729841e.png), OR
4. Minimally, the [support page](https://docs.nextcloud.com/server/19/user_manual/files/access_webdav.html) linked to in the above menu should describe the above solution, link to it, or *something*. 

### Describe alternatives you've considered
I cannot imagine I am the first person to think of this. I did  look for other issues describing the same thing but wasn't able to find. 

### Additional context
This would also make the app stuff more obvious and perhaps tempting to someone who is just using the web interface. It will remind them of how trivial it would be to set up. 

Nextcloud is being promoted as an alternative to google drive, etc. So I presume lots of people will want to install the app as a first order of business. Having the option of using a QR code (_extremely_ convenient) but then no way top *find* the QR code without digging into support forums made me feel really disappointed. It looks like a half-finished feature tbh.  I don't really even know what happens past the intro because I seem to have ended up on github instead of solving the problem I was working on..... Admittedly that's not exactly what will happen in every case. 🤓 But it might prompt something to think "I'll deal with this later" and never get to it.

I have heard good things about this app so of course will keep going with it. I am interested in a bit of learning along the way, but my impression is the nextcloud is being marketed to people who just want something that will *work* as well. 

I presume there are technical issues with generating QR codes everywhere the way I've described. If that's the case then there should be *links* to everything so it can be really smooth.

**Comment by tobiasKaminsky** [comment]
@nextcloud/server-triage 
@nextcloud/designers 

Indeed this is a workflow we should enhance.

**Comment by rullzer** [comment]
Yeah we know. There is also a server ticket about it. To add it to the firstrun wizard etc.
However the problem is that we need a new flow for that. As we otherwise would keep generating credentials all the time.

**Comment by tobiasKaminsky** [comment]
I could only find this one: https://github.com/nextcloud/server/issues/16469

**Comment by rullzer** [comment]
https://github.com/nextcloud/server/issues/16073

**Comment by CouldBeThis** [comment]
Not at all surprised this has been raised before. I have read the other threads which seems complementary.  

I am not a developer and I don't know what a *flow* is and wouldn't demand anyone explain it to me. Maybe the point in the request got lost in the lists and nesting and images and everything. To summarize I made 4 suggestions, which could be implemented in any combination as desired:

1. add help text to the app
2. change the text of a label on a button
3. put the QR code in the place that it was obvious (to me) to look for it; I think maybe this is what you are talking about? If it's extremely burdensome I would amend that proposal to instead link to the place you *can* get it
4. update documentation

So 3 of 4 are just changing text, and the fourth as modified would be just that. Not sure what it's about in the first run wizard but maybe this is being made into a bigger project than it has to be? (I don't know about anyone else but for myself I would like to get the server set up properly first before futzing around with a phone app anyway. I think for a novice user, encouraging them to start fiddling on their phone right during the install could be a bit distracting and confusing.) 

If indeed it's not possible to smooth things *at all*, I think taking the option *out* of the app (or moving it somewhere less prominent so it can be found by those who know where to look) should be considered because it really does detract from the first impression. It could read as either 

1. an incomplete feature is included in the final release of this app; I wonder what else isn't finished; should I trust it?  or
2. I am not smart enough to use this software; it is too complicated; I can't figure it out
